src/components/themed/SideMenu.tsx

```diff
@@ -152,6 +152,7 @@ export function SideMenuComponent(props: DrawerContentComponentProps) {
     const website = `${config.website}?af=appreferred_${refId}`
 
     const shareOptions = {
+      failOnCancel: false,
```

Comment:
Shouldn't we be showing the error in the catch for this `Share.open`?

Reply:
No. This prevents an error when the user simply cancelled sharing. Not an error at all.

Reply:
See other comment reply.

Reply:
ah ok. got it. will change